ketcher-autotests/tests/User-Interface/Editing-Tools/Copy-Cut-Paste/copy-cut-paste.spec.ts

```diff
@@ -0,0 +1,165 @@
+/* eslint-disable no-magic-numbers */
```

Comment:
Could you clarify, why did you disabled `no-magic-numbers` rule?

Reply:
Previously, I did for each line individually. But there were a lot of these lines, and in the comments to the pull request, they recommended me to do it for the entire file, since for each line it increases the number of lines. We use numbers to number atoms and bonds just like in the helper methods.
![2023-07-11_20h01_34](https://github.com/epam/ketcher/assets/62464241/3b5a245b-e79a-4007-8783-aa68c10252f8)